astroquery/eso/core.py

```diff
@@ -640,43 +642,78 @@ def _print_help(self, url, instrument, cache=True):
         resp = self._request("GET", url, cache=cache)
         doc = BeautifulSoup(resp.content, 'html5lib')
         form = doc.select("html body form pre")[0]
-        for section in form.select("table"):
```

Comment:
Taking out this part of the code and duplicating below is not the best coding practice. Maybe move the if statement above and define a variable there, and use it here to loop over?